client/src/actions/projects.ts

```diff
@@ -309,77 +305,95 @@ export const fetchApplicationStatusesFromContract =
             status: projectApplication.status,
           });
         }
-      } catch (error) {
+      } catch (error: any) {
         datadogRum.addError(error, { roundAddress });
         console.error("getApplicationStatusFromContract() error", {
```

Comment:
@DanieleSalatti do we need a console.error here if we report to ddog and dispatch the error?

Reply:
Replacing with a datadog logger